static/src/javascripts/projects/commercial/modules/header-bidding/prebid/price-config.ts

```diff
@@ -96,3 +96,113 @@ export const ozonePriceGranularity = (
 
 	return undefined;
 };
+
+export const indexPrebidPriceGranularity = (
+	width: number,
+	height: number,
+): PrebidPriceGranularity | undefined => {
+	const sizeString = [width, height].join(',');
+
+	if (
+		sizeString === adSizes.skyscraper.toString() ||
+		sizeString === adSizes.halfPage.toString()
+	) {
+		return {
+			buckets: [
+				{
+					max: 10,
+					increment: 0.01,
+				},
+				{
+					max: 15,
+					increment: 0.05,
+				},
+				{
+					max: 50,
+					increment: 1,
+				},
+			],
+		};
+	}
+
+	if (
+		sizeString === adSizes.leaderboard.toString() ||
+		sizeString === adSizes.billboard.toString() ||
+		sizeString === adSizes.mpu.toString()
+	) {
+		return {
+			buckets: [
+				{
+					max: 12,
+					increment: 0.01,
+				},
+				{
+					max: 20,
+					increment: 0.05,
+				},
+				{
+					max: 50,
+					increment: 1,
+				},
+			],
+		};
+	}
+
+	return undefined;
+};
+
+export const otherPrebidPriceGranularity = (
```

Comment:
is this just used in the test?

Reply:
It is for now. There will be another PR incoming where it will be used in the `prebid.ts` file when we run the script for the Prebid-other line items.